tools/lint/lint.py

```diff
@@ -844,8 +833,9 @@ def check_file_contents(repo_root, path, f=None):
     :param f: a file-like object with the file contents
     :returns: a list of errors found in ``f``
     """
-    with io.open(os.path.join(repo_root, path), 'rb') if f is None else nullcontext(f) as real_f:
-        assert real_f is not None  # Py2: prod mypy -2 into accepting this isn't None
+    if f is None:
+        f = io.open(os.path.join(repo_root, path), 'rb')
+    with f as real_f:
```

Comment:
I don't think ou need `with f as real_f` because I'm pretty sure all these context managers are just returning self in `__enter__`. So just `with f:` and then use f below.

Reply:
Feel free to pick this change into the other PR and fix if you want to get it landed soon. Heading out for a few hours...

Reply:
No actually I have 10 minutes. Fixing...

Reply:
Fixed.